packages/frontend/src/features/tableCalculation/components/TableCalculationModal.tsx

```diff
@@ -224,7 +227,22 @@ const TableCalculationModal: FC<Props> = ({
                             data={Object.values(TableCalculationType)}
                         ></Select>
                     </Tooltip>
-                    <Group position="apart">
+                </Stack>
+
+                <Stack
+                    sx={() => ({
+                        position: 'sticky',
+                        backgroundColor: 'white',
+                        bottom: 0,
+                        zIndex: 2,
+                    })}
+                >
+                    <Group
+                        position="apart"
+                        sx={() => ({
```

Comment:
most mantine components have most `style` properties as component props, so instead of dong `sx` yo ucould do `spacing={theme.spacing.md}`. afaik sx should only be used if the property is not directly available.

Reply:
Noted.

Reply:
Hi @rephus,
<Group spacing={theme.spacing.md}/>
spacing attribute if for spacing between the elements inside the group wrapper. [Ref](https://v6.mantine.dev/core/group/?t=props)
I wanted to add padding around the element <Group/> element. Hence, I used sx, since I wasn't able to adding padding otherwise.
Thanks :)